src/renderer/base/renderer.cpp

```diff
@@ -549,7 +542,7 @@ void Renderer::TriggerFontChange(const int iDpi, const FontInfoDesired& FontInfo
 // - centeringHint - The horizontal extent that glyphs are offset from center.
 // Return Value:
 // - <none>
-void Renderer::UpdateSoftFont(const gsl::span<const uint16_t> bitPattern, const SIZE cellSize, const size_t centeringHint)
+void Renderer::UpdateSoftFont(const gsl::span<const uint16_t> bitPattern, const til::size cellSize, const size_t centeringHint)

```

Comment:
you changed centeringHint to a `CoordType` (perhaps erroneously) in FontBuffer but not here

Reply:
I changed `FontBuffer`.